doc/releases/migration-guide-3.7.rst

```diff
@@ -99,6 +99,25 @@ Networking
   the long-duration zperf test show with correct throughput result.
   (:github:`69500`)
 
+* Each IPv4 address assigned to a network interface has an IPv4 netmask
+  tied to it instead of being set for the whole interface.
+  If there is only one IPv4 address specified for a network interface,
+  nothing changes from the user point of view. But, if there is more than
+  one IPv4 address / network interface, the netmask must be specified
```

Comment:
Would you like to mention this as one IPv4 address per network interface instead of `/` ?

Reply:
This section is not part of my changes. I think it is showing up here due to some rebase issue. Will rebase and push to see if this goes away.